tests/themes.retry

```diff
@@ -0,0 +1 @@
+localhost
```

Comment:
Wooops, error artefact...
Could you delete this one and eventuellay add tests/*.retry to gitignore ?